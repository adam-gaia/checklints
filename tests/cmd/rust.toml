bin.name = "checklist"
args = ["--no-user-checklists", "--no-cache", "--check", "../../../checklists"]
env.add = { RUST_LOG = "debug" }                                                # for debugging
