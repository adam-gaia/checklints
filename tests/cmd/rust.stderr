Summary:
Passed: 2, Failed: 0, Skipped: 0 (Total: 2) in [..]
